posthog/hogql_queries/experiments/test/test_trends_statistics_count.py

```diff
@@ -73,12 +73,12 @@ def run_test(stats_version, calculate_probabilities, are_results_significant, ca
             self.assertEqual(p_value, 1)
 
             # Control: ~10% conversion rate with wide interval due to small sample
-            self.assertAlmostEqual(intervals["control"][0], 0.055, places=2)  # Lower bound ~5.5%
-            self.assertAlmostEqual(intervals["control"][1], 0.182, places=2)  # Upper bound ~18.2%
+            self.assertAlmostEqual(intervals["control"][0], 0.055, delta=0.02)  # Lower bound ~5.5%
```

Comment:
this is unexpected to me, is it acceptable for this calculation to return 7.4% instead of 5.5%?

Reply:
@neilkakkar We're using the same test against both the first methodology and the second methodology. The two methodologies produce slightly different results (and the first methodology a higher degree of variability), so I've been working around that with `delta`.

But, it's a good question. I'm going to re-roll this PR and split out the assertions across all of the tests. Thanks for the review!